src/models/StudentGroup.ts

```diff
@@ -79,14 +94,31 @@ class StudentGroup {
     }
 
     @action
-    reset() {
-        this.name = this._pristine.name;
-        this.description = this._pristine.description;
+    removeStudent(student: User) {
+        return this.store.removeUser(this, student);
```

Comment:
I think this doesn't update the `adminIds`: If you remove an admin user from the group (not just withdraw admin privileges), that user is still displayed as an admin until you reload the UI. Maybe this isn't the right place to fix it, but it seems related :).

Reply:
Hmm, for Chrome/Edge it does... what browser are you usin? Is there an error message in the console?

Reply:
Indeed, thanks!